docs/client/cpdlc.md

```diff
@@ -4,36 +4,42 @@ title: CPDLC
 
 --8<-- "includes/abbreviations.md"
 
-In the real world, CPDLC operations are generally used by aircraft within airspaces in Class A airspace above `F245`.
+In the real world, CPDLC (Controller-Pilot Datalink Communications) is generally used by aircraft within Class A airspace above `F245`.
```

Comment:
I have added a CPDLC abbreviation now here https://github.com/vatpac-technology/sops/pull/312 . So the expanded bracket won't be necessary

Reply:
Excellent